changedetectionio/templates/settings.html

```diff
@@ -36,7 +36,7 @@
                         {{ render_field(form.base_url, placeholder="http://yoursite.com:5000/",
                         class="m-d") }}
                         <span class="pure-form-message-inline">
-                            Base URL used for the {base_url} token in notifications, default value is the ENV var 'base_url',
+                            Base URL used for the {base_url} token in notifications, default value is the ENV var 'base_url' (Currently "{{current_base_url}}"),
```

Comment:
Capitalisation of `base_url` is incorrect (should be `BASE_URL`)

Reply:
@jeremysherriff hey, not quite.. this is the token name, not the variable name  , see https://github.com/dgtlmoon/changedetection.io/blob/c55bf418c58ead942d603c752f1f247462561532/changedetectionio/templates/_common_fields.jinja#L41

Reply:
@jeremysherriff well spotted, thanks!!